src/components/card/konva-stage-viewer.tsx

```
@@ -0,0 +1,108 @@
+              [ElEMENT_TYPE.SOCIAL]: (
+                <SocialCanvasElement
+                  element={el as SocialElement}
+                  onDragEnd={() => {}}
+                  onDragMove={() => {}}
+                  onTransformEnd={() => {}}
+                  onSelect={() => {}}
+                  previewMode={previewMode}
+                />
+              ),
+            }}
+            default={<></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)